src/tests/ftest/io/ior_small.yaml

```diff
@@ -27,27 +27,41 @@ ior:
     client_processes:
         np_16:
             np: 16
-    api: DAOS
     test_file: daos:testFile
     repetitions: 2
     iorflags: !mux
         ssf:
-          F: "-v -W -w -r -R"
+          flags: "-v -W -w -r -R"
+          ior_api: !mux
+            daos:
+              api: DAOS
+            mpiio:
+              api: MPIIO
+          transfersize_blocksize: !mux
+            256B:
+              transfer_size: '256B'
+              block_size: '4M'
+            1M:
+              transfer_size: '1M'
+              block_size: '32M'
+
+          objectclass: !mux
+            oclass_SX:
+              daos_oclass: "SX"
+              mpiio_oclass: 214
```

Comment:
We should move away from specifying object class enumeration numbers and only use names.  The `daos_api.py` `get_object_class(number|string|class)` method can be used to convert the string into the appropriate python daos object class enumeration class and the `.value` of the enumeration class can be used to obtain the enumeration number.

Do we actually need `mpiio_oclass`? Can we just use the value in `daos_oclass` to set the `IOR_HINT__MPI__romio_daos_obj_class` env value?  
We can assign it like so (line 266 of src/tests/ftest/util/ior_utils.py):
`"IOR_HINT__MPI__romio_daos_obj_class": get_object_class("OC_{}".format(self.daos_oclass.value)).value`

The only unresolved issue would be the default.  In the current implementation the `IOR_HINT__MPI__romio_daos_obj_class` env is currently always set and has a default of `OC_SX`.  I see two options:
1.  If `daos_oclass` is not set don't assign the env
2.  Default `daos_oclass` to `SX`

Reply:
A slightly more fool proof way of doing things - if someone still used a number for `doas_oclass`:

```
"IOR_HINT__MPI__romio_daos_obj_class":
    get_object_class("OC_{}".format(get_object_class(self.daos_oclass.value).name)).value
```